shared/src/main/scala/minitest/runner/Framework.scala

```diff
@@ -25,13 +25,15 @@ class Framework extends BaseFramework {
 
   def options: Options = Options()
 
-  def fingerprints(): Array[Fingerprint] =
+  // overriding method of type (): Array[sbt.testing.Fingerprint | JavaNull] | JavaNull
+  def fingerprints(): Array[Fingerprint | Null] =
```

Comment:
**Override java method**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.

Number of occurrences in file: 3